src/vector/vector_tests/mod.rs

```diff
@@ -491,4 +491,30 @@ mod tests {
         assert_eq!(ft.field("Value").unwrap().unwrap().into_real(), Some(45.78));
         assert_eq!(ft.field("Int_value").unwrap().unwrap().into_int(), Some(1));
     }
+
+    #[test]
+    fn test_features_reset() {
+        with_layer("roads.geojson", |mut layer| {
+            assert_eq!(layer.features().count(), layer.features().count(),);
+        });
+    }
+
+    #[test]
+    fn test_features_aliasing_compile_fail() {
+        let t = trybuild::TestCases::new(); // A compilation test that should fail.
+        t.compile_fail("tests/ui/01-features-aliasing-errors.rs");
```

Comment:
why is the tests folder called "ui"?

Reply:
Not sure;  I just followed the instructions from the the trybuild crate docs.  Not having the `ui/` folder caused some errors as (I think) cargo tried to run them as normal tests, and failed.  I didn't try a different folder name though.

Reply:
It seems that any folder inside `tests/` works;  have renamed it to `compile-tests`